coderd/healthcheck/workspaceproxy.go

```diff
@@ -78,6 +79,7 @@ func (r *WorkspaceProxyReport) Run(ctx context.Context, opts *WorkspaceProxyRepo
 	})
 
 	var total, healthy int
+	errs := make([]string, 0)
```

Comment:
nit: I never know which form is better. Yours or `var errs []string`.

Reply:
I suppose this form is better if you know exactly how many elements there are to be in the slice, which we don't.